src/main/java/com/auctionapp/api/controller/AuctionController.java

```diff
@@ -33,4 +35,11 @@
 
 		return ResponseEntity.status(HttpStatus.OK).body(auctions);
 	}
+
+	@GetMapping("/{uuid}")
+	public ResponseEntity<AuctionDto> get(@PathVariable UUID uuid) {
```

Comment:
All method parameters shoud be `final`, as well as everything else that can be. The param name in this method should be `auctionId` or simply `id`, `uuid` is a type name.